frontend/src/queries/nodes/InsightViz/PropertyGroupFilters/PropertyGroupFilters.tsx

```diff
@@ -54,11 +54,27 @@ export function PropertyGroupFilters({
         <div className="space-y-2 PropertyGroupFilters">
             {propertyGroupFilter.values && (
                 <BindLogic logic={propertyGroupFilterLogic} props={logicProps}>
-                    <InsightTestAccountFilter
-                        disabledReason={disabledReason}
-                        query={query}
-                        setQuery={setQuery as (node: InsightQueryNode) => void}
-                    />
+                    <div className="flex flex-1 gap-2 flex-row space-between">
```

Comment:
Ah yeah - I commented on this screenshot.

What's the rationale for this change? In the screenshot examples it forces the label for InsightTestAccountFilter onto two lines which IMO looks weird

Reply:
I will defer to @thmsobrmlr and @PostHog/team-product-analytics on this :)